pkg/controller/backup_sync_controller.go

```diff
@@ -141,9 +143,10 @@ func (c *backupSyncController) deleteUnused(cloudBackupNames sets.String) {
 		return
 	}
 
-	// For each backup object in Kubernetes, verify if has a corresponding backup in the object storage. If not, delete it.
+	// For each backup object in Kubernetes, verify if it is complete and
```

Comment:
this comment is slightly confusing/misleading. sounds like all non-completed backups get deleted.  I'd say `for each completed backup object in Kubernetes, delete it if it does not have a corresponding backup in object storage`

Reply:
Fixing..